tests/common.rs

```diff
@@ -0,0 +1,41 @@
+use axum::Router;
+use blockfrost_platform::{
+    cli::{Config, LogLevel, Mode, Network},
+    server::build,
+    AppError, NodePool,
+};
+use lazy_static::lazy_static;
+use std::sync::{Arc, Once};
+use tower_http::normalize_path::NormalizePath;
+
+lazy_static! {
+    static ref INIT: Once = Once::new();
+}
+
+pub fn initialize_logging() {
+    INIT.call_once(|| {
+        tracing_subscriber::fmt::init();
+    });
+}
+
+pub fn test_config() -> Arc<Config> {
+    let config = Config {
+        server_address: "0.0.0.0".into(),
+        server_port: 8080,
+        log_level: LogLevel::Info.into(),
+        network_magic: 2,
+        mode: Mode::Compact,
+        node_socket_path: "/run/cardano-node/node.socket".into(),
```

Comment:
Maybe default to that, but allow overriding with an env. variable? E.g. I have it locally in a different location, and had to edit sources to run the tests locally. A variable named `CARDANO_NODE_SOCKET_PATH` is commonly used.